src/uu/install/src/install.rs

```diff
@@ -52,14 +48,179 @@ impl Behavior {
     }
 }
 
+static ABOUT: &str = "Copy SOURCE to DEST or multiple SOURCE(s) to the existing
```

Comment:
My local `install` manual  says...

```text
In the first three forms, copy SOURCE to DEST or multiple SOURCE(s) to the existing
DIRECTORY, while setting permission modes and owner/group. In the 4th form, create
all components of the given DIRECTORY(ies).
```

I'm not sure how best to represent that here, but I'm good with this as a starting point.
```

Reply:
most of the features are missing in this program, so, still a lot of work+tests :)